components/api_calls/build_calls.js

```diff
@@ -6,14 +6,18 @@ export async function getBuilds(searchParams) {
     for (const key in searchParams) {
         let value = searchParams[key];
 
-        if ((value[0] === "\"" && value[value.length - 1] === "\"") || (value[0] === "'" && value[value.length - 1] === "'")) {
-            value = value.substring(1, value.length - 1);
+        // Check if the value is enclosed in double or single quotes using regex.
+        const isEnclosedInQuotes = /^["'].*["']$/.test(value);
```

Comment:
So `.*` is something that is best avoided, as a good practice. We will probably only have alpha numerals in addition to `-`. I think taking some time to think about which all values a variable can have or match is always worth it.

Reply:
ohh I had no idea, it makes sense thanks for the advice.